Sentinel/Classes/Core/CustomInfoTool.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  CustomInfoTool.swift
+//  Sentinel
+//
+//  Created by Vlaho Poluta on 30/07/2020.
+//
+
+import Foundation
+
+public class CustomInfoTool: Tool {
```

Comment:
we should add objc support 😄

Reply:
Added objc support to CustomInfoTool, CustomLocationTool, TextEditingTool and UserDefaultsTool